install.sh

```diff
@@ -80,7 +80,10 @@ check_platform() {
 }
 tag_to_version() {
   if [ -z "${TAG}" ]; then
```

Comment:
What happens if this gets run as 

```
./install.sh latest
```

and there is a v2 release in this repo? I think it will try to download the v2 release and fail.

Should we do something like this instead, or is that too surprising?

```
if [ -z "${TAG}" ] || [ "${TAG}" == "latest" ]; then
```